GoogleSignIn/Sources/Public/GoogleSignIn/GIDSignIn.h

```diff
@@ -139,7 +141,8 @@ typedef void (^GIDDisconnectCallback)(NSError *_Nullable error);
 ///     called asynchronously on the main queue.
 - (void)addScopes:(NSArray<NSString *> *)scopes
     presentingViewController:(UIViewController *)presentingViewController
-                    callback:(nullable GIDSignInCallback)callback;
+                    callback:(nullable GIDSignInCallback)callback
+    NS_EXTENSION_UNAVAILABLE("The add scopes flow is not supported in App Extensions."); 
 
 /// Marks current user as being in the signed out state.
 - (void)signOut;
```

Comment:
Does it make sense that `-signOut` and `-disconnectWithCallback:` are available?

Reply:
Technically, those methods are available.  Whether or not they make sense to use in an extension, I'm thinking we leave to developers.

Reply:
Sounds like the issue at stake here doesn't impact these methods (since they do not use the `sharedApplication` to do their work). I'm resolving.